integration/jnifuse/fs/src/main/java/alluxio/jnifuse/AbstractFuseFileSystem.java

```diff
@@ -72,10 +73,11 @@ public void mount(boolean blocking, boolean debug, String[] fuseOpts) {
     if (mountPointStr.endsWith("\\")) {
       mountPointStr = mountPointStr.substring(0, mountPointStr.length() - 1);
     }
+    String maxIdleThreadsStr = "max_idle_threads=" + maxIdleThreads;
     if (!debug) {
-      arg = new String[] {getFileSystemName(), "-f", mountPointStr};
+      arg = new String[] {getFileSystemName(), "-f", mountPointStr, maxIdleThreadsStr};
     } else {
-      arg = new String[] {getFileSystemName(), "-f", "-d", mountPointStr};
+      arg = new String[] {getFileSystemName(), "-f", "-d", mountPointStr, maxIdleThreadsStr};
```

Comment:
@ssz1997 no need to change on the JNIFuse side?

Reply:
From my testing on linux there's no need to change JNIFuse, and I can see more than 10 threads in WAITING or TIMED_WAITING stage when I run stressbench.